vm/sdk/src/call.rs

```diff
@@ -12,5 +20,25 @@ pub fn call(address: Address, input: Option<Vec<u8>>, method: Option<&str>, amou
   let input_payload = payload.encode();
   let input_len = input_payload.len();
   let input = crate::bytes_to_u32_vec(input_payload);
-  athena_vm::syscalls::call(address.as_ptr(), input.as_ptr(), input_len, amount.as_ptr());
+
+  // Allocate 1024 bytes on the heap, using u32 for proper alignment
+  let mut output_buf = Vec::<std::mem::MaybeUninit<u32>>::with_capacity(1024 / 4);
+
+  let output_size = athena_vm::syscalls::call(
+    address.as_ptr(),
+    input.as_ptr(),
+    input_len,
+    output_buf.as_mut_ptr() as *mut u32,
+    1024,
```

Comment:
```suggestion
    1024, // TODO: may be changed, see https://github.com/athenavm/athena/issues/177
```

Reply:
Considering this value is used in multiple places, can we make this a const and define it only once? E.g., it's also used on line 25 above

Reply:
Done